src/extras/audio/Audio.js

```diff
@@ -17,12 +17,15 @@ THREE.Audio = function ( listener ) {
 	this.panner = this.context.createPanner();
 	this.panner.connect( this.gain );
 
+	this.startTime = 0;
+	this.isPlaying = false;
+
 };
 
 THREE.Audio.prototype = Object.create( THREE.Object3D.prototype );
 THREE.Audio.prototype.constructor = THREE.Audio;
 
-THREE.Audio.prototype.load = function ( file ) {
+THREE.Audio.prototype.load = function ( file, playImmediately ) {
```

Comment:
I think I would mimic the HTML5 audio a bit more and add the boolean `autoplay` (default to `false`) as property of `THREE.Audio`.